### PR TITLE
L10n: Add basic translation instructions

### PR DESCRIPTION
**Pull request by Xexxa**

### CODE REVIEW

File: l10n.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Add your .po-file to [/po/CMakeLists.txt](https://github.com/bugaevc/ladybird-gtk4/blob/master/po/CMakeLists.txt)
+```
+    set(linguas
+        eo.po
+        ru.po
+        sv.po
+    )
+```
+### 3. org.serenityos.Ladybird-gtk4.desktop
+Translate the strings in [org.serenityos.Ladybird-gtk4.desktop](https://github.com/bugaevc/ladybird-gtk4/blob/master/org.serenityos.Ladybird-gtk4.desktop)
```
Comment (bugaevc) [inline l10n.md]: Link to https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#localized-keys here somewhere

File: l10n.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Translating Adwaita components
+
+Some components in Ladybird, like `Tab Overview` ([TabOverview](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.TabOverview.html)) and `About Ladybird` ([AboutWindow](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.AboutWindow.html)) uses components from Adwaita. These might or might not already have a translation, you can find these in [https://gitlab.gnome.org/GNOME/libadwaita/-/tree/main/po](https://gitlab.gnome.org/GNOME/libadwaita/-/tree/main/po).
+
+
+## Resources for translating
+
+GNOME Wiki might have information about localization & translating your language, give `https://wiki.gnome.org/{Language name in English}` a look. (e.g. https://wiki.gnome.org/Esperanto, https://wiki.gnome.org/Swedish).
+
+If libadwaita is already translated to your language then feel free to reuse terms and words for consistency with Adwaita/GNOME https://gitlab.gnome.org/GNOME/libadwaita/-/tree/main/po
```
Comment (bugaevc) [inline l10n.md]: When in doubt, it's also a good idea to look at how the similar component / string is translated in Epiphany. We don't follow Epiphany 100%, but it's also a very handy resource for design & translation ideas.

File: l10n.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Translating Ladybird
+## gettext
+Ladybird is translated with the help of [gettext](https://www.gnu.org/software/gettext/).
+
+Add a new translation:
+
+### 1. Add a new po file
+
+You can find current translations in https://github.com/bugaevc/ladybird-gtk4/tree/master/po, copy one of the files and translate all the strings `msgstr` **[what is the command the user should run instead of copying one of the current files?]**
```
Comment (bugaevc) [inline l10n.md]: What I do is create just an empty file (or you can fill in the header at this point), add it to `linguas`, and run the build once:

```
$ touch ../po/foo.po
$ nano ../po/CMakeLists.txt
$ ninja
(does stuff)
```

that will "update" (really, generate) the PO file with all the strings to translate.

Comment (Xexxa) [reply l10n.md]: Couldn't get it to work so I'm leaving it as is for other contributors to improve upon!

Comment (Xexxa) [reply l10n.md]: "Allow edits by maintainers" is on if you want to have a go at it. Or the PR can be closed if it's not good enough. 

Can't really explain the ninja stuff ;-)

Comment (bugaevc) [reply l10n.md]: Don't worry, I haven't forgotten about this PR 🙂, just haven't gotten around to completing the doc.

Comment (bugaevc) [reply l10n.md]: Done :)

**Pull request merged**